homeassistant/components/remote/harmony.py

```diff
@@ -152,9 +155,7 @@ class HarmonyRemote(remote.RemoteDevice):
 
     def __init__(self, name, host, port, activity, out_path, delay_secs):
         """Initialize HarmonyRemote class."""
-        from aioharmony.harmonyapi import (
-            HarmonyAPI as HarmonyClient, ClientCallbackType
-        )
+        from aioharmony.harmonyapi import HarmonyAPI as HarmonyClient
 
         _LOGGER.debug("%s: Device init started", name)
```

Comment:
We try to avoid side effects in init. We can remove this log. We log in `async_added_to_hass` too.

Reply:
Check failed due to another issue not related to my PR, hence to kick it off again had to do a new push. That _LOGGER line entry is removed. :-)

Build passed now, ready to be merged :-)